addons/voice/org.openhab.voice.marytts/src/main/java/org/openhab/voice/marytts/internal/MaryTTSAudioStream.java

```diff
@@ -89,7 +89,7 @@ private InputStream getWavHeaderInputStream(int length) throws IOException {
         byte format = 0x10; // PCM
         byte bits = 16;
         byte channel = 1;
-        int srate = 48000;
+        long srate = (this.audioFormat != null) ? this.audioFormat.getFrequency() : 16000l;
```

Comment:
Why do you use 16000l as a default now instead of 48000l? Imho, the default should make sure that at least the English version works correctly.